src/Inkluzitron/Modules/ImagesModule.cs

```diff
@@ -128,6 +128,7 @@ private async Task<RuntimeResult> DomSubRolledImageAsync(IUser target, bool show
                 if (!check.IsSuccessful)
                 {
                     await PointsService.AddPointsAsync(Context.User, check.SubstractedPoints, decrement:true);
+                    await PointsService.AddPointsAsync(target, check.SubstractedPoints);
```

Comment:
Z mobilu nevidím kontext a vím že jsem to psal původně ja, ale nešlo by přičítání/odečítání bodů přesunout až za odeslání obrázku? Ať to zbytečně neblokuje a neprodlužuje čekání na gif

Reply:
Mám obavu, že paralelizace z toho udělá guláš :(